monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunOptions.js

```diff
@@ -73,15 +72,7 @@ function RunOptions(props) {
                                setComponent(LocalManualRunOptions,
                                  {ips: ips, setComponent: setComponent})
                              }}/>
-        {shouldShowScoutsuite(props.islandMode) && <AWSRunButton setComponent={setComponent}/> }
-        {shouldShowScoutsuite(props.islandMode) && <NextSelectionButton title={'Cloud security scan'}
-                             description={'Explains how to enable cloud security scan.'}
-                             icon={faCloud}
-                             onButtonClick={() => {
-                               setComponent(CloudOptions,
-                                 {ips: ips, setComponent: setComponent})
-                             }}/>
-        }
+        {isNotRansomware(props.islandMode) && <AWSRunButton setComponent={setComponent}/> }
```

Comment:
`isModeRansomware` or `showRunOnAwsButton` are better names IMO, not important though